<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUserWindow</class>
 <widget class="QMainWindow" name="AddUserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Animeplex - Add User Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 50, 50)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="AddUserWindow_MainFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>901</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	border-image: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="AddUserWindow_BackBtnFrame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="AddUserWindow_BackBtn">
          <property name="minimumSize">
           <size>
            <width>95</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>95</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0);
	color: white;	
	border: none;
}

QPushButton::hover {
	background-color: rgb(186, 0, 0);
}</string>
          </property>
          <property name="text">
           <string>â®œ   BACK</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="AddUserWindow_TextFrame">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>641</width>
       <height>181</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="AddUserWindow_HeadingLabel">
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>32</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: white;
}</string>
          </property>
          <property name="text">
           <string>Create/Add A User.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="AddUserWindow_TextLabel">
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: white;
}</string>
          </property>
          <property name="text">
           <string>To login, you must have an account. Below you need to enter the basic information to create an account.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="AddUserWindow_InputsFrame">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>340</y>
       <width>591</width>
       <height>231</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="AddUserWindow_FieldLabel_1">
            <property name="font">
             <font>
              <family>Noto Sans</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
	color: white;
}</string>
            </property>
            <property name="text">
             <string>Enter Email Address:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="AddUserWindow_EmailLineEdit">
            <property name="minimumSize">
             <size>
              <width>340</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>340</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Noto Sans</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(27, 27, 27);
	color: white;
	border: none;
}</string>
            </property>
            <property name="placeholderText">
             <string>Ex: burner@gmail.com</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="AddUserWindow_FieldLabel_2">
            <property name="font">
             <font>
              <family>Noto Sans</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
	color: white;
}</string>
            </property>
            <property name="text">
             <string>Create A Password:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="AddUserWindow_PasswordLineEdit">
            <property name="minimumSize">
             <size>
              <width>340</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>340</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Noto Sans</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(27, 27, 27);
	color: white;
	border: none;
}</string>
            </property>
            <property name="placeholderText">
             <string>Ex: password123</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="AddUserWindow_FieldLabel_3">
            <property name="font">
             <font>
              <family>Noto Sans</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
	color: white;
}</string>
            </property>
            <property name="text">
             <string>Re-enter Password:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="AddUserWindow_ConfirmPasswordLineEdit">
            <property name="minimumSize">
             <size>
              <width>340</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>340</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Noto Sans</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(27, 27, 27);
	color: white;
	border: none;
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="AddUserWindow_BtnsFrame">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>620</y>
       <width>591</width>
       <height>121</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="AddUserWindow_ClearBtn">
          <property name="minimumSize">
           <size>
            <width>145</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0);
	color: white;	
	border: none;
}

QPushButton::hover {
	background-color: rgb(186, 0, 0);
}</string>
          </property>
          <property name="text">
           <string>CLEAR INPUTS</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="AddUserWindow_CreateBtn">
          <property name="minimumSize">
           <size>
            <width>135</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(18, 18, 18);
	color: white;
	border: none;
}

QPushButton::hover {
	background-color: rgb(140, 4, 24);
}</string>
          </property>
          <property name="text">
           <string>ADD/CREATE</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="AddUserWindow_InvalidEmailLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>785</y>
       <width>571</width>
       <height>0</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: rgb(204, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>Error: Invalid Email Address</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="AddUserWindow_PasswordsDontMatchLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>780</y>
       <width>571</width>
       <height>0</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: rgb(204, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>Error: Passwords do not match</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="AddUserWindow_AccountAlreadyExistsLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>780</y>
       <width>571</width>
       <height>0</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: rgb(204, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>Error: That user already exsists</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="AddUserWindow_SuccessLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>780</y>
       <width>571</width>
       <height>0</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: rgb(0, 190, 0);
}</string>
     </property>
     <property name="text">
      <string>Successfully created user!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="AddUserWindow_EmptyInputsLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>780</y>
       <width>571</width>
       <height>0</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: rgb(204, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>Error: Empty inputs</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="AddUserWindow_SpacesDetectedLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>780</y>
       <width>571</width>
       <height>0</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: rgb(204, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>Error: Spaces detected</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
